app/helpers/admin/orders_helper.rb

```diff
@@ -11,9 +11,26 @@ module OrdersHelper
     def order_adjustments_for_display(order)
       order.adjustments +
         voucher_included_tax_representations(order) +
+        additional_tax_total_representation(order) +
         order.all_adjustments.payment_fee.eligible
     end
 
+    def additional_tax_total_representation(order)
+      adjustment = Spree::Adjustment.where(order_id: order.id, included: false,
+                                           originator_type: 'Spree::TaxRate',
+                                           adjustable_type: 'Spree::Adjustment')
+        .sum(:amount)
```

Comment:
`Spree::Adjustment` has a bunch of `scopes` defined so it's easier to write and understand such queries: 
https://github.com/openfoodfoundation/openfoodnetwork/blob/15d4f9565d730d038119c5de1ae9375c7b0aef49/app/models/spree/adjustment.rb#L65-L85
So should be able to rewrite is like this : 
```suggestion
      adjustment = Spree::Adjustment.additional.tax.where(
        order_id: order.id, adjustable_type: 'Spree::Adjustment'
      ).sum(:amount)
```

Reply:
thanks for the tip Gaetan, i have made the changes